gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystem.java

```diff
@@ -427,6 +476,48 @@ public void delete(URI path, boolean recursive) throws IOException {
     repairImplicitDirectory(parentInfoFuture);
   }
 
+  private String getBucketName(URI path) {
```

Comment:
won't path.getAuthority() give the bucketname?

Reply:
Updated. Missed it from my end !